gen/definitions/system.yaml

```diff
@@ -586,6 +586,9 @@ attributes:
     tf_name: standby_redirects_enable_disable
     example: disable
     exclude_test: true
+  - yang_name: ip/routing-new/routing/protocol/purge/interface
```

Comment:
After making changes to the yaml file, you would need to run "go generate" to update the code, documentation and test cases.